### ISSUE TITLE
Jira integration

### ISSUE THREAD

**Issue by twindan** [alexhajdu/fix_this#9]
It would be great if this would directly integrate for Jira.   The basic integration could just be to associate the bug number and then have some way of launching the web browser with the bug from the game.  

A more advanced integration could require custom properties to be defined in Jira, such as the map name and location. The plugin could then push that data to Jira (via a "sync" button) so someone looking at the bug would know which map/location to go to.  Closing an issue could also close it in Jira.

**Comment by alexhajdu** [comment]
Yes, trello integration is like 90% ready. The 2 ways communication is already implemented. I plan to implement Jira, Assembla, Hacknplan...I'm really not limited.
My idea was to take and use bare minimum all ticketing systems are having in common. It's a ticket, list, board, member, label. So I'm using description part of ticket as a holder of data I need for 2 ways communication.

![image](https://user-images.githubusercontent.com/1824796/55587958-e09bdf80-572c-11e9-8e5b-7b55c3cad2fa.png)
![image](https://user-images.githubusercontent.com/1824796/55588007-00330800-572d-11e9-9080-d3c9d48af508.png)
